tests/lib/rules/no-array-prototype-extensions.js

```diff
@@ -576,9 +594,9 @@ import { get as g } from 'dummy';
       errors: [{ messageId: 'main', type: 'CallExpression' }],
     },
     {
-      code: 'something.somethingElse.without(1)',
+      code: "something.somethingElse.without('abc')",
       output:
-        'something.somethingElse.indexOf(1) > -1 ? something.somethingElse.filter(item => item !== 1) : something.somethingElse',
+        "something.somethingElse.indexOf('abc') > -1 ? something.somethingElse.filter(item => item !== 'abc') : something.somethingElse",
```

Comment:
I just realized that we need to wrap this ternary expression with parenthesis due to the below test case. Can you fix that here or in a separate PR?

```js
something.without(1).foo();
```

We can rely on prettier to remove any unnecessary parenthesis after the autofix.

Reply:
It's a good idea to try the autofixer on a real codebase to catch issues like this.

Reply:
Great catch. Will fix it as part of this PR itself.